(Wx) Use tabs for create/recent on main

Desbeers opened the pull request:
Looks better than radio buttons!

![Screenshot from 2024-11-02 17-07-01](https://github.com/user-attachments/assets/5293ad55-5b07-4f0c-ac43-149a20402d18)

sciurius commented:
IIRC we've tried notebook tabs before but somehow changed to radio buttons. 

Can you please change the panel names according to the conventions so far (i.e. `p_create` instead of `Create` etc.)?

Desbeers commented:
Renamed the panels and also removed the empty toolbar because of Windows....

Last time we tried something with notebooks it gave a problem with programatically adding/removing/selecting them if I remember correctly. Here they are just static and seems to work fine.
